packages/code-du-travail-frontend/src/modules/enterprise/EnterpriseAgreementSearch/EnterpriseAgreementSearchInput.tsx

```diff
@@ -0,0 +1,238 @@
+      </p>
+      <form
+        className={fr.cx(
+          "fr-grid-row",
+          "fr-grid-row--top",
+          "fr-mt-2w",
+          "fr-mb-0"
+        )}
+        onSubmit={async (event) => {
+          event.preventDefault();
+          if (!search) {
+            setSearchState("noSearch");
+            return;
+          }
+          setLoading(true);
+          try {
+            const result = await searchEnterprises({
+              query: search,
+              apiGeoResult: location,
+            });
+            setSearchState(!result.length ? "errorSearch" : "fullSearch");
+            setSearchState(
+              search.length > 1 && !result.length
+                ? "notFoundSearch"
+                : "noSearch"
+            );
+            setEnterprises(result);
+          } catch (e) {
+            setSearchState("errorSearch");
+            setEnterprises(undefined);
+            setError(e);
+          } finally {
+            setLoading(false);
+            setSearched(true);
+          }
+        }}
+      >
+        <Input
+          className={fr.cx("fr-col-12", "fr-col-md-5", "fr-mb-0")}
+          hintText={
+            <>
+              Ex : Café de la mairie ou 40123778000127
+              <br />
+              (présent sur la fiche de paie du salarié)
+            </>
+          }
+          label={<>Nom de votre entreprise ou numéro Siren/Siret</>}
+          state={getInputState()}
+          stateRelatedMessage={getStateMessage()}
+          nativeInputProps={{
+            onChange: (event) => {
+              setSearch(event.target.value);
+            },
+          }}
+          classes={{
+            label: css({
+              "& > button": {
+                padding: "0!",
+                minHeight: "0!",
+                height: "20px!",
+                width: "24px!",
+                marginLeft: "3px!",
+              },
+            }),
+          }}
+        />
+        <LocationSearchInput
+          onLocationChange={setLocation}
+          className={fr.cx(
+            "fr-col-12",
+            "fr-col-md-3",
+            "fr-ml-md-3w",
+            searched && !enterprises?.length ? "fr-mb-7w" : "fr-mb-0",
+            "fr-mt-2w",
+            "fr-mt-md-0"
+          )}
+          classes={{ label: fr.cx("fr-mb-md-7v") }}
+        />
+
+        <Button
+          type="submit"
+          iconPosition="right"
+          iconId="fr-icon-search-line"
+          className={`${fr.cx("fr-ml-md-3w", "fr-mt-2w", "fr-mt-md-19v", searched && !enterprises?.length ? "fr-mb-7w" : "fr-mb-0")} ${ButtonStyle}`}
+        >
+          Rechercher
+        </Button>
+      </form>
+
+      <div>
+        <div className={fr.cx("fr-mt-2w")}>
+          {!!enterprises?.length && !loading && (
+            <p className={fr.cx("fr-h5")}>
+              {enterprises.length} entreprises trouvées
+            </p>
+          )}
+          {loading && <p className={fr.cx("fr-h5")}>chargement en cours ...</p>}
```

Comment:
c'est subjectif mais je trouve ce texte très gros j'aurai mis un `<p>` normal
```suggestion
          {loading && <p>chargement en cours ...</p>}
```